src/core/reference/include/openvino/reference/subtract.hpp

```diff
@@ -13,9 +13,17 @@
 
 namespace ov {
 namespace reference {
+namespace func {
+// Usage of custom function instead of std::minus gives smaller binary size.
+template <class T>
+constexpr T subtract(const T a, const T b) {
+    return a - b;
+}
+}  // namespace func
```

Comment:
what about performance speed?

Reply:
@rkazants are you ok with the answer and can we process it?